buildSrc/src/main/java/io/github/tarek360/dependencies/MainApp.kt

```diff
@@ -0,0 +1,9 @@
+package io.github.tarek360.dependencies
+
+object MainApp {
```

Comment:
This is cool but the name`MainApp` makes me confused, why `MainApp` is there another type of apps or what?
I think it's better to rename this object to `Project`

FYI:
`io.github.tarek360.app.App.kt` is a sample app to explain how can anyone use Koshry but I think after merging your PR, the `app` module should be deleted because we have `buildSrc` now so we can create a Gradle task with Koshry dependency to use Koshry inside Koshry.

Reply:
I am already making a object class called Projects that has the names of the modules we used, so, we can make this class Modules and the MainApp class we can make it Project then. So, what's your opinion ?

Reply:
Yes, that's better.